app/services/data_migrations/backfill_enic_reason.rb

```diff
@@ -0,0 +1,16 @@
+module DataMigrations
+  class BackfillEnicReason
+    TIMESTAMP = 20240729111251
+    MANUAL_RUN = true
+
+    def change
+      ApplicationQualification.where.not(enic_reference: nil).in_batches do |batch|
```

Comment:
```suggestion
      ApplicationQualification.where(enic_reason: nil).where.not(enic_reference: nil).in_batches do |batch|
```
Probably overkill but is safe incase we need to rerun the migration